doc/datasets/index.rst

```diff
@@ -171,7 +171,7 @@ near-equal-size classes separated by concentric hyperspheres.
 :func:`make_circles` and :func:`make_moons` generate 2d binary classification
 datasets that are challenging to certain algorithms (e.g. centroid-based
 clustering or linear classification), including optional Gaussian noise.
-They are useful for visualisation. produces Gaussian
+They are useful for visualisation. Produces Gaussian
```

Comment:
this seems strange,  maybe ``..., and to produce...``?

Reply:
I agree, but wasn't sure on what the message was trying to convey so I went with the smallest change possible. And to produce would seem to convey the same message and be clearer.

Reply:
Only `make_circles` produces a spherical decision function so the sentence does not apply to both.

How about:

> :func:`make_circles` produces Gaussian data with a spherical decision boundary for binary classification, while :func:`make_moons` produces two interleaving half circles.

Reply:
Yeah definitely thought maybe that there might be a different meaning because the sentence seemed to be unfinished. I changed it and changed the formatting by adding backquotes.

> :func:`make_circles ... :func:`make_moons` ...

This was to match with other references in the documentation, please tell me if that shouldn't be the case in this instance. Thank you